server/api/admin/categories.js

```diff
@@ -21,28 +21,39 @@ router.delete('/delete', async (req, res) => {
 // Update category with a specific category ID
 router.patch('/:categoryID', async (req, res) => {
   if ('name' in req.body) {
-    await Category.findByIdAndUpdate(req.params.categoryID, {
-      name: req.body.name
-    }, (err, category) => {
-      if (err) res.status(500).send(err);
-    });
+    await Category.findByIdAndUpdate(
+      req.params.categoryID,
+      {
+        name: req.body.name,
+      },
+      (err) => {
+        if (err) res.status(500).send(err);
+      },
+    );
   }
 
   if ('companyName' in req.body) {
-    await Category.findByIdAndUpdate(req.params.categoryID, {
-      companyName: req.body.companyName
-    }, (err, category) => {
-      if (err) res.status(500).send(err);
-    });
+    await Category.findByIdAndUpdate(
+      req.params.categoryID,
+      {
+        companyName: req.body.companyName,
+      },
+      (err) => {
+        if (err) res.status(500).send(err);
+      },
+    );
   }
 
   if ('type' in req.body) {
-    await Category.findByIdAndUpdate(req.params.categoryID, {
-      type: req.body.type
-    }, (err, category) => {
-      if (err) res.status(500).send(err);
-      
-    });
+    await Category.findByIdAndUpdate(
+      req.params.categoryID,
+      {
+        type: req.body.type,
+      },
+      (err) => {
+        if (err) res.status(500).send(err);
+      },
+    );
```

Comment:
Rather than updating everything in 3 if statements, can you do it in a single statement?

I would take a look at Mongoose's [`save` ](https://mongoosejs.com/docs/documents.html#updating-using-save) function:
1. Retrieve the document from db with `doc = Model.findById()`
2. Modify the document with changes
3. Save the document in the db with `doc.save()`